### PR TITLE
Fix/stabilize tests

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/UserAndGroupManagement.java [hunk 1]
```diff
@@ -455,6 +455,10 @@ public void deleteMembership(String membershipName, boolean verify) {
    */
   public void goToEditUserInfo(String username) {
     info("--Search user " + username + "--");
+    if($(byTitle("Quick Search")).is(Condition.not(Condition.exist)))
+    {
+      $(byXpath("//*[@id=\"UIOrganizationPortlet\"]/div[2]/div[1]/ul/li[1]/a")).click();
+    }
```
Comment (thomasdelhomenie) [inline gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/UserAndGroupManagement.java]: Selectors should be moved to constants.

File: gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/UserAndGroupManagement.java [hunk 2]
```diff
@@ -455,6 +455,10 @@ public void deleteMembership(String membershipName, boolean verify) {
    */
   public void goToEditUserInfo(String username) {
     info("--Search user " + username + "--");
+    if($(byTitle("Quick Search")).is(Condition.not(Condition.exist)))
+    {
```
Comment (thomasdelhomenie) [inline gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/UserAndGroupManagement.java]: Bracket must be at the end of the line.

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java [hunk 3]
```diff
@@ -288,9 +289,10 @@ public void addDrives(String name, String permission, specificView[] applyViews)
    *          other will be uncheck)
    */
   public void editDrives(String name, specificView[] applyViews, String... views) {
-    $(byClassName("actionIcon")).click();
+    executeJavaScript("window.scrollBy(0,-250);");
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: Just curious, the element is not clickable if you do not scroll ?

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java [hunk 4]
```diff
@@ -329,8 +331,7 @@ public void editDrives(String name, specificView[] applyViews, String... views)
    * @param name
    */
   public void deleteDrives(String name) {
-   // evt.click(By.xpath(ELEMENT_ECM_EXPLORER_DRIVES_DELETE_LIST.replace("{$name}", name)));
-    $(byTitle("Delete")).click();
+$(byText(name)).parent().parent().find((byTitle("Delete"))).click();
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: Indentation is not good.

File: wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/RichTextEditor.java [hunk 5]
```diff
@@ -39,6 +41,7 @@ public RichTextEditor(TestBase testBase) throws Exception {
     this.evt = testBase.getElementEventTestBase();
     this.but = new Button(testBase);
     this.plf = new PlatformBase(testBase);
+    this.wikiManagement=new WikiManagement(testBase);
```
Comment (thomasdelhomenie) [inline wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/RichTextEditor.java]: Add spaces around =

File: wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/RichTextEditor.java [hunk 6]
```diff
@@ -519,12 +522,18 @@ public void goToEditLink() {
    */
   public void addSimplePage(String title, String content) {
     info("Input a title for the page");
+    $(ELEMENT_TITLE_WIKI_INPUT).waitUntil(Condition.appears,Configuration.timeout);
+    if($(ELEMENT_SOURCE_EDITOR_BUTTON).is(Condition.not(Condition.exist))&&(ELEMENT_BUTTON_WIKI_RITCH_TEXT.is(Condition.exist)))
+    {ELEMENT_BUTTON_WIKI_RITCH_TEXT.click();}
```
Comment (thomasdelhomenie) [inline wiki/src/main/java/org/exoplatform/platform/qa/ui/wiki/pageobject/RichTextEditor.java]: Indentation is not good.

File: wiki/src/test/java/org/exoplatform/platform/ui/qa/wiki/WikiBasicActionAddRichTextTestIT.java [hunk 7]
```diff
@@ -88,6 +92,8 @@ public void test01_AddAPageWithLinkWikiPageExisted() {
     String tooltip = "tooltip" + getRandomNumber();
     homePagePlatform.goToWiki();
     wikiHomePage.goToAddBlankPage();
+    if($(ELEMENT_CONTENT_WIKI_INPUT).is((Condition.exist)))
+    {ELEMENT_BUTTON_WIKI_RITCH_TEXT.click();}
```
Comment (thomasdelhomenie) [inline wiki/src/test/java/org/exoplatform/platform/ui/qa/wiki/WikiBasicActionAddRichTextTestIT.java]: Indentation is not good.

File: wiki/src/test/java/org/exoplatform/platform/ui/qa/wiki/WikiPublishActivityTestIT.java [hunk 8]
```diff
@@ -53,6 +55,7 @@ public void setupBeforeMethod() {
    * <li>Post-Condition:</li>
    */
   @Test
+  @Tag("test")
```
Comment (thomasdelhomenie) [inline wiki/src/test/java/org/exoplatform/platform/ui/qa/wiki/WikiPublishActivityTestIT.java]: What is the purpose of this tag ?

File: gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/UserAndGroupManagement.java [hunk 9]
```diff
@@ -80,8 +80,8 @@ public void goToUsersTab() {
    */
   public void goToGroupTab() {
     info("-- Choose Group Management tab--");
-    $(byText("Groups")).click();
-    $(byText("Group Info")).waitUntil(Condition.appears, Configuration.timeout);
+    ELEMENT_GROUP_TAB.click();
```
Comment (ilyeskallala) [inline gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/UserAndGroupManagement.java]: yes, I will change the element to "byxpath"

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java (hunk 3 above)
Comment (ilyeskallala) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/administration/ContentAdministration.java]: yes, we put Y as negative value for scroll up and positive value for scroll down (Y is the number of pixels )

File: wiki/src/test/java/org/exoplatform/platform/ui/qa/wiki/WikiPublishActivityTestIT.java (hunk 8 above)
Comment (ilyeskallala) [inline wiki/src/test/java/org/exoplatform/platform/ui/qa/wiki/WikiPublishActivityTestIT.java]: I inserted this tag to run just one test case (which is under this tag) with maven and i forgot to remove it

**Comment by mgreau** [comment]
PR merged

**Pull request closed without merging**